plugins/inputs/procstat/spec_processor.go

```diff
@@ -102,7 +102,7 @@ func (p *SpecProcessor) pushMetrics() error {
 		fields[prefix+"cpu_time_guest_nice"] = cpu_time.GuestNice
 	}
 
-	cpu_perc, err := p.proc.Percent(time.Duration(0))
+	cpu_perc, err := p.proc.Percent(time.Duration(3000 * time.Millisecond))
```

Comment:
It can be few milliseconds. However 3 seconds would be a good duration for getting correct cpu_usage percentage. 1ms would be too less for calculating to cpu_percentage. I can make it 1s